src/interfaces/ResolverFilterData.ts

```diff
@@ -8,10 +8,3 @@ export interface ResolverFilterData<TPayload = any, TArgs = ArgsDictionary, TCon
   context: TContext;
   info: GraphQLResolveInfo;
 }
-
-export interface ResolverTopicData<TPayload = any, TArgs = ArgsDictionary, TContext = {}> {
```

Comment:
I mean that you should just create a reference `export type ResolverTopicData = ResolverFilterData` for now, maybe in the future it will be distinguishable.